src/pages/create/index.tsx

```diff
@@ -1,16 +1,51 @@
 import { AppWrapper } from 'components/common/CoreAppWrapper/CoreAppWrapper'
 import { Head } from 'components/common/Head/Head'
 import { CV_V3 } from 'constants/cv'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
+import { OPEN_IPFS_GATEWAY_HOSTNAME } from 'constants/ipfs'
+import { NETWORKS_BY_NAME } from 'constants/networks'
 import { SiteBaseUrl } from 'constants/url'
 import { TransactionProvider } from 'contexts/Transaction/TransactionProvider'
-import { Create } from 'packages/v2v3/components/Create/Create'
+import { JBProjectProvider } from 'juice-sdk-react'
+import { NetworkName } from 'models/networkName'
+import { Create as V2V3Create } from 'packages/v2v3/components/Create/Create'
 import { V2V3ContractsProvider } from 'packages/v2v3/contexts/Contracts/V2V3ContractsProvider'
 import { V2V3CurrencyProvider } from 'packages/v2v3/contexts/V2V3CurrencyProvider'
+import { Create as V4Create } from 'packages/v4/components/Create/Create'
+import { wagmiConfig } from 'packages/v4/wagmiConfig'
 import { Provider } from 'react-redux'
 import store from 'redux/store'
+import { featureFlagEnabled } from 'utils/featureFlags'
 import globalGetServerSideProps from 'utils/next-server/globalGetServerSideProps'
+import { WagmiProvider } from 'wagmi'
 
-export default function V2CreatePage() {
+export default function CreatePage() {
+  let contentByVersion = (
+    <V2V3ContractsProvider initialCv={CV_V3}>
+      <TransactionProvider>
+        <V2V3CurrencyProvider>
+          <V2V3Create />
+        </V2V3CurrencyProvider>
+      </TransactionProvider>
+    </V2V3ContractsProvider>
+  )
+
+  if (featureFlagEnabled(FEATURE_FLAGS.V4)) {
+    contentByVersion = (
+      <WagmiProvider config={wagmiConfig}>
+        {/* Hardcode V4 create to Sepoila for now */}
+        <JBProjectProvider
+          chainId={NETWORKS_BY_NAME[NetworkName.sepolia].chainId as 11155111}
```

Comment:
Sets the network for useJBContractContext to use. Ideally we would read from `process.env.NEXT_PUBLIC_INFURA_NETWORK`, but I think this will require us to first update `jbDirectoryAddress` in the SDK to include mainnet (chainId=1).

<img width="783" alt="Screenshot 2024-08-29 at 10 18 59 am" src="https://github.com/user-attachments/assets/5e75056d-da64-4309-91aa-51e11de76053">